dev_scripts/generate_output_files.py

```diff
@@ -0,0 +1,276 @@
+    if 'coord' in test_definition:
+        coord_fpath = os.path.join(gendir, test_definition['coord'])
+    else:
+        coord_fpath = os.path.join(gendir, 'coord')
+
+    shutil.copy(coord_fpath, os.path.join(rundir, 'coord'))
+
+
+exec_to_out_obj = dict(exec_to_out_obj)
+exec_to_out_obj['jobex'] = JobexOutput
+
+
+def main():
+    """Main function"""
+    parser = argparse.ArgumentParser()
+    args = get_args(parser)
+
+    if args.list:
+        return print_tests()
+
+    tests_list = get_tests_list(args, parser)
+    dryrun = args.dryrun
+    version_dir, ref_version_dir = get_version_dir(args)
+
+    for tm_exec, test_name in tests_list:
+        test_dir = os.path.join(version_dir, tm_exec, test_name)
+        test_run_dir = os.path.join(test_dir, 'run')
+        ref_test_dir = os.path.join(ref_version_dir, tm_exec, test_name)
+        gen_test_dir = os.path.join(gen_dir, tm_exec, test_name)
+        deftest_fpath = os.path.join(gen_test_dir, "test.yaml")
+        if not os.path.isfile(deftest_fpath):
+            raise RuntimeError(f'No test.yaml file for test {tm_exec}/{test_name}')
+        deftest = loadfn(deftest_fpath)
+        if deftest.get('disable', False):
+            print(f'!Generation of outputs for test {tm_exec}/{test_name} is disabled')
+            continue
+
+        print(f'Generation of outputs for test {tm_exec}/{test_name}')
+
+        # Create run directory for test and copy coord file
+        makedirs_p(test_run_dir)
+        get_coord(test_definition=deftest, rundir=test_run_dir, gendir=gen_test_dir)
+
+        with cd(test_run_dir):
+            all_diffs = {}
+            if args.generate_control:
+                if deftest['define'] is None:
+                    raise ValueError('Cannot generate control file from define parameters (control file is fixed).')
+                generate_control_for_test(test_definition=deftest)
+                if dryrun:
+                    ref_control_fpath = os.path.join(
+                        gen_test_dir, deftest['control']
+                        if 'control' in deftest else 'control'
+                    )
+                    ref_control = Control.from_file(ref_control_fpath)
+                    test_control = Control.from_file(os.path.join(test_run_dir, 'control'))
+                    control_diffs = test_control.compare(ref_control, return_all_diffs=True)
+                    if control_diffs:
+                        msg = 'There are differences in the control files generated:\n'
+                        for idiff, diff in enumerate(control_diffs, start=1):
+                            msg += f'#{idiff} {diff}\n'
+                        if args.print_diffs:
+                            print(msg)
+                        all_diffs['control'] = control_diffs
+            else:
+                if deftest['define'] and deftest['define']['template']:
+                    dp = get_define_template('ridft')
```

Comment:
rereading this comment I have one question: the templates are the default in the input/templates folder, so what happens if these get changed? It may happen for a variety of reasons (options added or removed in TM, more suitable values as defaults, ...). Maybe it is safer to make a copy in the testfile folder, that could be kept more under control? 

We can even decide to do this in the future if the templates actually get changed.

Reply:
Fixed 'ridft' to deftest['define']['template'].
For the templates, let's keep that for the future if/when the templates get changed.